gcp/admin-server/main.tf

```diff
@@ -44,7 +44,6 @@ sudo dnf install -y google-cloud-cli-gke-gcloud-auth-plugin 2>> $log
   metadata          = {}
   resource_policies = []
   tags              = []
-  depends_on        = [google_project_service.compute]
```

Comment:
I don't think this is strictly necessary.

Reply:
how does this dependency get calculated without this line?

Reply:
It's not necessary.